packages/xfsprogs/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "xfsprogs"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "/dev/null"
+
+[package.metadata.build-package]
+releases-url = "https://xfs.wiki.kernel.org"
```

Would this be more accurate?
```suggestion
releases-url = "https://mirrors.edge.kernel.org/pub/linux/utils/fs/xfs/xfsprogs/"
```

---

Yeah, that is probably more accurate. I was thinking of this more as the `URL` in the specfile but strictly speaking the releases are found at that URL.